data/mods.json

```diff
@@ -1,261 +1,297 @@
-      "versions": {
-        "11.15.1.2318": "596512ad5f12f95d8a3170321543d4455d23b8fe649c68580c5f828fe74f6668"
-      }
-    },
-    "Essential": {
+    "essential": {
+      "name": "Essential",
       "download": "https://essential.gg/",
       "versions": {
         "1.0.0-13428": "a70c02554a96faff60647eb204652a20b5551798506f1e4c0970daef401f782e"
       }
     },
-    "OneConfig": {
+    "oneconfig": {
+      "name": "OneConfig",
       "download": "https://github.com/Polyfrost/OneConfig",
       "versions": {
         "0.2.1-alpha181": "004fc41df0eaa5e5e5cdc1e50f66f3b7b85ab85b03b453898db97726e43dc472"
       }
     },
     "OneCore by OneConfig": {
+      "name": "OneCore by OneConfig",
       "download": "https://github.com/Polyfrost/OneConfig",
       "versions": {
         "1.3.4": "7b0a95bbe06a49c3acf3fb63dccc5c65b5317f4b912477245dea24d52de605f2"
       }
     },
     "ChatTriggers": {
+      "name": "ChatTriggers",
       "download": "https://www.chattriggers.com/",
       "versions": {
         "2.2.0": "3b077d3e99eb714d13e76826d75929807e8ab582023c59bc405d4bad67193254"
       }
     },
     "NotEnoughUpdates": {
+      "name": "NotEnoughUpdates",
       "download": "https://github.com/NotEnoughUpdates/NotEnoughUpdates",
       "versions": {
         "2.1.1+Alpha-23.e9c3bbb": "945e90438a1e9925d8ea4dbe5ecd8efa5bd3f57d71953bbe2855c158df8903e00"
       }
     },
     "Patcher": {
+      "name": "Patcher",
       "download": "https://sk1er.club/mods/patcher",
       "versions": {
         "1.8.6": "4a134770967bf18c91345d39f70f49373adfcbce9a4adc8d3f4116c44f1ad3a9",
         "1.8.7": "0c61939fbd472f96c02c40bd77ef447c5719fecbb52c0a40eb90aaf198e7dd34"
       }
     },
     "SkyblockAddons": {
+      "name": "SkyblockAddons",
       "download": "https://github.com/BiscuitDevelopment/SkyblockAddons",
       "versions": {
         "1.7.0": "d78b8b7d8e11bf3dcfa6bc0e11ff5828ae160f5fa67de4bd344cbb7b91cf2fb4"
       }
     },
     "SkyHanni": {
+      "name": "SkyHanni",
       "download": "https://github.com/hannibal002/SkyHanni",
       "versions": {
         "0.20": "9053dfb1b572f96283bcf461bdb09e8779dcfe67d291c558585e86a28f4349cf",
         "0.21.Beta.12": "f7514c894fe647873026d5e227327b181ccbfef7be433828b6eaeacdb1378395"
       }
     },
     "Skytils": {
+      "name": "Skytils",
       "download": "https://github.com/Skytils/SkytilsMod",
       "versions": {
         "1.7.8": "bed2f299f5acebc4c1fbbc1a753d351a518b1ce8944be87403db73ff298fc778"
       }
     },
     "SoopyV2Forge": {
+      "name": "SoopyV2Forge",
       "download": "https://github.com/Soopyboo32/SoopyV2",
       "versions": {
         "1.1": "9623d49290b41f9d271bc3c53a69a48e77eda323614e92fc73ba288ff3a648f3"
       }
     },
     "spark": {
+      "name": "Spark",
       "download": "https://github.com/nea89o/spark",
       "versions": {
         "1.10.37+nea": "477a2ecf0b53e2d9d9d5b620a2d7177e81c4204055126e608d8e89d9647887be"
       }
     },
     "Dulkir Mod": {
```

Comment:
why are some ids from uppercase to lowercase and some not

Reply:
I stopped halfway through and decided I'm gonna do this with the discord bot automatically

Reply:
Ah I see